app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
It's good practice to use type annotations. Here, the `cafe` parameter is missing a type annotation. You should add one to make your code more clear and maintainable.